buildpipeline/windows.groovy

```diff
@@ -15,57 +15,79 @@ simpleNode('Windows_NT','latest') {
     }
 
     def logFolder = getLogFolder()
+    def framework = ''
+    if (params.TargetGroup == 'all') {
+        framework = '-allConfigurations'
+    }
+    else {
+        framework = '-framework:${params.TargetGroup}'
+    }
+    def archGroup = 'x64' // Should parameterize in the future.
+
+    def submitToHelix = (params.TargetGroup == 'netcoreapp')
 
     stage ('Clean') {
         bat '.\\clean.cmd -all'
     }
     stage ('Sync') {
-        bat '.\\sync.cmd -p -- /p:ArchGroup=x64 /p:RuntimeOS=win10'
+        bat '.\\sync.cmd -p -- /p:ArchGroup=${archGroup} /p:RuntimeOS=win10'
     }
     stage ('Generate Version Assets') {
         bat '.\\build-managed.cmd -GenerateVersion'
     }
     stage ('Build Product') {
-        bat ".\\build.cmd -buildArch=x64 -${params.Config} -- /p:SignType=real /p:RuntimeOS=win10"
+        bat ".\\build.cmd ${framework} -buildArch=${archGroup} -${params.Config} -- /p:SignType=real /p:RuntimeOS=win10"
     }
     stage ('Build Tests') {
         def additionalArgs = ''
+        def archiveTests = 'false'
         if (params.OuterLoop) {
-            additionalArgs = '-Outerloop'
+            additionalArgs += '-Outerloop'
+        }
+        if (submitToHelix) {
+            additionalArgs += '-SkipTests'
+            archiveTests = 'true'
+        }
+        if (params.TargetGroup == 'uap' || params.TargetGroup == 'uapaot') {
```

Comment:
Right now the conditions are mutually exclusive but your point is taken I will fix these to make sure we don't add SkipTests multiple times in the future.